lib/workers/repository/reconfigure/index.spec.ts

```diff
@@ -0,0 +1,157 @@
+import { mock } from 'jest-mock-extended';
+import {
+  RenovateConfig,
+  fs,
+  git,
+  mocked,
+  platform,
+  scm,
+} from '../../../../test/util';
+import { logger } from '../../../logger';
+import type { Pr } from '../../../modules/platform/types';
+import * as _cache from '../../../util/cache/repository';
+import { validateReconfigureBranch } from '.';
+
+jest.mock('../../../util/cache/repository');
+jest.mock('../../../util/fs');
+jest.mock('../../../util/git');
+
+const cache = mocked(_cache);
+
+describe('workers/repository/reconfigure/index', () => {
+  const config: RenovateConfig = {
+    branchPrefix: 'prefix/',
+    baseBranch: 'base',
+  };
+
+  beforeEach(() => {
+    config.repository = 'some/repo';
+    scm.getFileList.mockResolvedValue([]);
+    scm.branchExists.mockResolvedValue(true);
+    cache.getCache.mockReturnValue({});
+    git.getBranchCommit.mockReturnValue('sha');
+    fs.readLocalFile.mockResolvedValue(null);
+    platform.getBranchPr.mockResolvedValue(null);
+  });
+
+  it('no effect on repo with no reconfigure branch', async () => {
+    scm.branchExists.mockResolvedValueOnce(false);
+    await validateReconfigureBranch(config);
+    expect(logger.debug).toHaveBeenCalledWith('No reconfigure branch found');
+  });
+
+  it('throws error if config file not found', async () => {
```

Comment:
```suggestion
  it('throws error if config file not found in reconfigure branch', async () => {
```

The `logger.warn` message says this is about the config file in the reconfigure branch. I suppose we also have _another_ test _somewhere else_ that is a generic "check if there's a Renovate config file at all in the repository".